### PR TITLE
Fix the camelcase filter issue which doesn't treat a space

### PR DESCRIPTION
**Pull request by hwanseung**
Fix the camelCase filter issue which doesn't treat a space.
 - Fix the camelCase issue
 - Replace camelCase with pascalCase
   (We are used upper camel case and that is called pascalCase commonly.)

ISSUE=#168

### CODE REVIEW

File: generator/main.ts [hunk 1]
```diff
@@ -14,20 +14,18 @@
...
-import * as reader from './reader/simple_reader';
-
-import snakeCase = require('snake-case');
-
 import EnumTypes from './parser/enum_types';
 import IDLDefinition from './parser/idl_definition';
 import IDLEnum from './parser/idl_enum';
 import IDLInterface from './parser/idl_interface';
 import Parser from './parser/parser';
+import * as reader from './reader/simple_reader';
```
Comment (hwanseung) [inline generator/main.ts:28]: i don't want to move this line.
but our format command moved this line.

**Comment by hwanseung** [comment]
um... travis CI tool has some problem at os x environment.
it seems internal error of travis.
`fatal: unable to access 'https://github.com/lunchclass/bacardi.git/': Could not resolve host: github.com
The command "eval git clone --depth=50 https://github.com/lunchclass/bacardi.git lunchclass/bacardi " failed. Retrying, 2 of 3.`
https://travis-ci.org/lunchclass/bacardi/jobs/299615214

**Comment by yjaeseok** [comment]
lgtm !!

File: generator/main.ts (hunk 1 above)
Comment (romandev) [reply generator/main.ts:28]: Ack; The clang will sort the module string(e.g. ./reader/simple_reader) order.

File: generator/main.ts [hunk 2]
```diff
@@ -104,13 +102,11 @@ async function main([root_dir, out_dir, ...idl_files]) {
   });
 
   var env = new nunjucks.Environment();
-  env.addFilter('camelcase', function(str, count) {
-    return str.replace(/(?:^\w|[A-Z]|\b\w|\s+)/g, function(match, index) {
-      return index == 0 ? match.toUpperCase() : match;
-    });
+  env.addFilter('pascalcase', function(str, count) {
```
Comment (romandev) [inline generator/main.ts]: Is this possible?
`env.addFilter('pascalcase', changeCase.pascalCase);`
(In javascript, I'm sure it works but I'm not sure in TypeScript)

**Review by romandev — COMMENTED** [review]
overall looks good to me.
Please check bot error. (I didn't open yet)

**Review by romandev — COMMENTED** [review]
Additionally, we don't use '.' in commit title.

File: generator/main.ts (hunk 2 above)
Comment (hwanseung) [reply generator/main.ts]: it is working well.
so i replaced it.

**Comment by hwanseung** [comment]
i removed period at title.

File: generator/main.ts (hunk 1 above)
Comment (hwanseung) [reply generator/main.ts:28]: okay i see.

**Pull request merged**